docs/detections/machine-learning/machine-learning.asciidoc

```diff
@@ -76,6 +76,19 @@ prior to the time they are enabled. After jobs are enabled, they continuously
 analyze incoming data. When jobs are stopped and restarted within the two-week
 time frame, previously analyzed data is not processed again.
 
+[float]
+[[ml-integrations]]
+=== Jobs in advanced analytics (UEBA) Elastic integrations
+
+{ml} jobs can also be installed via https://docs.elastic.co/integrations[Elastic integrations]. We currently have the following Advanced Analytics integrations for Security:
```

Comment:
Change `machine learning` to uppercase.

Reply:
I'll check about this, they have an existing definition used throughout the ML docs (the `{ml}` includes tag) which is lowercase, so I currently use that one for consistency with other pages 
But good to double check!

Reply:
Oh, wait, it was lowercase despite being the first word in the sentence, I'll fix this 😁 Thank you!